Reports/ProgressReport-Feb24-Aug24.md

```diff
@@ -0,0 +1,318 @@
+- Created a new website, not launched yet.
+- Created a public community calendar, for "official" events and all those happening in the space and that people may want to include and promote.
+
+We celebrated the March quarterly event with very positive feedback from participants. Keynote was delivered by NHS England R&D programme Director on the subnaitonal SDEs programme, this followed a request by community members to know more about it during the December event. Full notes can be found here while we produce a report https://hackmd.io/6t-s4x-wQN6riL6gPJLuWg
+
+With funding ending we received a community request/push to ensure we put the workshop budget to good use. In response we elaborated two events proposals that align with the goal of this grant: finalising governance docs with the community participation and set up WGs for success.
+- Documentation sprints (https://hackmd.io/ty_bEhtBTrKjmYsJEEP8Qw)
+- Working groups day (https://hackmd.io/nTAAo7fOT_quE18y0NYjnA). Which will be held on 29 April (This also implied figuring out a way to spend beyond March given it is not feasible to run the WGs day effectively before late April).
+
+Lastly we have been workign with Scriberia to use the remaining funding to produce an illustration that can represent the community in a number of ways, the most immediate one being the production of physical stand up banners for events (due to timeline DSH will cover the actual banner while DARE funds have covered the illustration itself).
+
+
+#### Communication and outreach [#35](https://github.com/alan-turing-institute/data-safe-haven-team/issues/35)
+
+Contributes to:
+- Creating and maintaining open and active communication spaces & workspaces (Slack, GH)
+- Creating resources for all stakeholders (inc. Citizens) to engage in the TRE conversation
+
+##### Goal
+
+- Supporting the user community of the DSH codebase
+- Publicising our work via blogposts, reports or papers
+- Communicating our work through conference/workshop talks or posters 
+
+##### Progress
+
+AI UK has been a priority for this story and the project. Held on 19-20 March DSH had its own stand, through a likert scale exercise on TREs (and a bowl of sweets) we engaged attendees to introduce them to TREs and the work of the project which we stressed to be not only an open codebase but also the governance & standards and the community.
+
+The stand was busy on the first day and quieter on the second, in part due to its location out of sight from the rest, but that left time to properly engage in conversation with those who did (NHS England, Head of cyber-physica & digital twins at Innovate UK, Chief Exec of RSS). Overall the impression of those there has been that there was important potential leads, we have added to our private sharepoint for follow up.  
```

Comment:
It might be worth expanding on this. I definitely felt that I had fewer conversations, but those that I had were more detailed and friendly. I also felt they were more likely to lead to useful connections or collaboration.

Reply:
Yes, I added just a bit but would be happy for all of us there to add to this